### PR TITLE
PostContent: ensure group feed spacing correct

### PR DESCRIPTION
**Pull request by arthyn**
Refactors post content height detection to use resize observer so we always get correct truncation and adds small margin when doing so.

### CODE REVIEW

**Review by liam-fitzgerald — COMMENTED** [review]
Have you checked this on safari? It's a total madhouse vis a vis resize observer. Else, LGTM!

**Comment by arthyn** [comment]
Checked Safari, get this 
image

but it's still working, and from what I can tell by googling that issue, doesn't seem to affect anything.

File: pkg/interface/src/views/landscape/components/Home/Post/PostItem/PostContent.tsx [hunk 1]
```diff
@@ -27,15 +29,13 @@ const PostContent = ({ post, isParent }: PostContentProps): ReactElement => {
...
   const showFade = !isParent && height >= 300;
 
-  const measuredRef = useCallback((node) => {
-    if (node !== null) {
-      setHeight(node.getBoundingClientRect().height);
-    }
-  }, []);
+  const resizer = useResize<HTMLDivElement>(useCallback((entry) => {
+    setHeight(entry.target.getBoundingClientRect().height);
+  }, [setHeight]));
```
Comment (liam-fitzgerald) [inline pkg/interface/src/views/landscape/components/Home/Post/PostItem/PostContent.tsx:34]: setters unnecessary in dependency lists

**Review by liam-fitzgerald — APPROVED** [review]
Nit, else LGTM

**Pull request merged**